WooJeong/refactor/Torch2Circle.py

```diff
@@ -0,0 +1,72 @@
+import os
+import shutil
+import sys
+
+import onnx
+import onnx_tf
+import tensorflow as tf
+import torch
+import torch.nn
+import torch.quantization
+
+sys.path.append('./include')
+import subprocess
+
+#  generated by pics.
+#  we need to set dependency on cmakelist
+from include.circle.Model import Model
+
+
+class Torch2Circle:
+
+    @staticmethod
+    def toCircle(original_model: torch.nn.Module, sample_input: torch.Tensor, dir_path: str,
+                 tflite2circle_path='./tflite2circle', clean_circle=True):
+        if not os.path.exists(tflite2circle_path):
+            raise Exception('tflite2circle not exists')
+        tmp_path = os.path.join(dir_path, 'tmp')
+        if not os.path.exists(tmp_path):
+            os.makedirs(tmp_path, exist_ok=True)
+        tmp_path = os.path.join(dir_path, 'tmp')

```

Comment:
tmp_path is duplicated. Is there any reason for this?

Reply:
During refactoring in rush, might be forgot to remove duplicated variable.
I'll change it right now